spyder/plugins/ipythonconsole/comms/tests/test_comms.py

```diff
@@ -102,8 +102,14 @@ def comms(kernel):
     frontend_comm = FrontendComm(kernel)
     kernel_comm = KernelComm()
 
+    def dummy(port):
```

Comment:
Please change the name of this function to something more meaningful. I mean, I don't understand what this `dummy` is doing below.

Reply:
This function is not doing anything. I don't use real comms in this test so it would fail if I try to set a port.

Reply:
Yeah, but I don't understand it's purpose. Could you give it a better name so it's clearer why it's necessary here?

Reply:
done